### DISCUSSION TITLE
Shairport-server Response Code: 500 to Pipewire sends "POST /feedback RTSP/1.0"

### DISCUSSION THREAD

**Discussion by alexbramford** [mikebrady/shairport-sync#1744] [Q&A]
- PipeWire version: 0.3.81
- Distribution and distribution version: "Fedora Linux 38 (Workstation Edition)"
- Desktop Environment: KDE 5.110.0 / Plasma 5.27.8
- Kernel version: 6.5.5-200.fc38.x86_64

## Description of Question:
Why does Shairport respond with "Response Code: 500" when Pipewire sends `POST /feedback` to raop-sink?

1. Pipewire sends `POST /feedback RTSP/1.0" to the Shairport-server`
2. Shairport responds with `Response Code: 500`
3. Pipewire reports `status: RTSP/1.0 500 Internal Server Error`

## How Reproducible:
Always

### Steps to Reproduce:
Restart client and server, play music
1. Server side (Shairport-sync): `sudo systemctl restart shairport-sync; systemctl status shairport-sync`
2. Client side (F38): `killall easyeffects; systemctl --user daemon-reload; sleep 1; PIPEWIRE_DEBUG=0 systemctl --user restart pipewire pipewire-pulse  wireplumber`
3. Play music
4. `journalctl  -f` (client and server)

### Actual Results:
------------------------------------------------------------------------------------------------------------------------
Pipewire

```
Oct 06 14:24:30 pipewire[28560]: default: sent: POST /feedback RTSP/1.0
                                 CSeq: 581
                                 Client-Instance: 96FB269ED974ED15
                                 DACP-ID: 96FB269ED974ED15
                                 User-Agent: PipeWire/0.3.81
                                 Session: 1
Oct 06 14:24:30 pipewire[28560]: default: status: RTSP/1.0 500 Internal Server Error
Oct 06 14:24:30 pipewire[28560]: default:  CSeq: 581
Oct 06 14:24:30 pipewire[28560]: default:  Server: AirTunes/105.1
Oct 06 14:24:30 pipewire[28560]: default: received reply to request with cseq:581
Oct 06 14:24:30 pipewire[28560]: mod.raop-sink: reply status: 500
```

------------------------------------------------------------------------------------------------------------------------
Shairport:
```
Oct 06 14:24:30 shairportserver shairport-sync[8732]:          0.031028148 "rtsp.c:5182" Connection 2: Received an RTSP Packet of type "POST":
Oct 06 14:24:30 shairportserver shairport-sync[8732]:          0.000065018 "rtsp.c:5184"   Type: "CSeq", content: "581"
Oct 06 14:24:30 shairportserver shairport-sync[8732]:          0.000017241 "rtsp.c:5184"   Type: "Client-Instance", content: "96FB269ED974ED15"
Oct 06 14:24:30 shairportserver shairport-sync[8732]:          0.000015074 "rtsp.c:5184"   Type: "DACP-ID", content: "96FB269ED974ED15"
Oct 06 14:24:30 shairportserver shairport-sync[8732]:          0.000014037 "rtsp.c:5184"   Type: "User-Agent", content: "PipeWire/0.3.81"
Oct 06 14:24:30 shairportserver shairport-sync[8732]:          0.000016204 "rtsp.c:5184"   Type: "Session", content: "1"
Oct 06 14:24:30 shairportserver shairport-sync[8732]:          0.000015166 "rtsp.c:5184"   No Content Plist. Content length: 0.
Oct 06 14:24:30 shairportserver shairport-sync[8732]:          0.000016760 "rtsp.c:2042" Connection 2: POST /feedback Content-Length 0
Oct 06 14:24:30 shairportserver shairport-sync[8732]:          0.000014851 "rtsp.c:5243" Connection 2: RTSP Response:
Oct 06 14:24:30 shairportserver shairport-sync[8732]:          0.000014334 "rtsp.c:5244"   Response Code: 500.
```

### Expected Results:
What is the purpose of pipewire sending `POST /feedback` ?

# Additional Info (as attachments):
Raised for pipewire perspective: https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/3549

[pipewire.conf.txt](https://github.com/mikebrady/shairport-sync/files/12835961/pipewire.conf.txt)

[shairport-sync.conf.txt](https://github.com/mikebrady/shairport-sync/files/12835962/shairport-sync.conf.txt)

**Comment by mikebrady** [comment]
Thanks for the interesting port, but this is really a troubleshooting issue and could use some more information from the Shairport Sync end of things...

I've just posted it as an [issue](https://github.com/mikebrady/shairport-sync/issues/1745).
